sgx-crypto/src/lib.rs

```diff
@@ -90,13 +102,19 @@ impl ToArray for bn::BigNumRef {
     }
 }
 
+/// When implemented for Signature, this trait signs the MRENCLAVE value created
```

Comment:
I'm not sure what "implemented for Signature" means. The `trait Signature` can be implemented for *something else*.

Reply:
This is purely an extension trait for the `Signature` struct. This trait could theoretically be implemented on other types. However, it doesn't make much sense.

Reply:
I updated the language to be a bit more specific; hopefully it works.